crates/sui-core/src/authority_server.rs

```diff
@@ -332,7 +332,7 @@ impl ValidatorService {
         let tx_verif_metrics_guard = metrics.tx_verification_latency.start_timer();
         let transaction = epoch_store
             .signature_verifier
-            .verify_tx(transaction.data())
+            .verify_tx(transaction.data(), Some(state.get_google_jwk_as_bytes()))
```

Comment:
can we instead push the new jwk to the signature verifier whenever it changes, e.g.

```
signature_verifier.update_jwk(new_jwk)
```

Reply:
you are suggesting to put the jwk updater inside AuthorityPerEpochStore (that has the signature_verifier) instead of AuthorityState?